ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -373,6 +383,10 @@ Resources:
           Value: "yes"
         - Key: "PatchGroup"
           Value: "prod-default"
+    CreationPolicy:
+      ResourceSignal:
+        Count: 7 #Must equal the number of calls to cfn-signal during provisioning
```

Comment:
I actually think we only need 1 signal, the one from `UserData:`.  That signal is sent after all init sections have completed.

Reply:
Confirmed